.github/workflows/python-ci.yml

```diff
@@ -5,32 +5,37 @@ on: push
 jobs:
   test:
 
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7, 3.8]
```

Comment:
What are our criteria for choosing which versions of Python to test with? Why 3.7? Why not 3.9? Asking for a friend :)

Reply:
Usually I try to keep the version I use on my machine and a slightly older versions that others may be using. It actually may be a good idea to test and support all version that should be compatible with the project. In this case I think we could continue testing `3.6`.